core/templates/pages/about-page/about-page.mainpage.html

```diff
@@ -1,26 +1,10 @@
 <!DOCTYPE html>
-<html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
+<html itemscope itemtype="http://schema.org/Organization">
   <head>
     @load('base-components/header.template.html', {"title": "About | Oppia"})
   </head>
   <body>
-    <oppia-root>
-      <div ng-controller="Base" dir="<[direction]>">
-        <oppia-base-content>
-          <navbar-breadcrumb>
-            <ul class="nav navbar-nav oppia-navbar-breadcrumb">
-              <li>
-                <span class="oppia-navbar-breadcrumb-separator"></span>
-                <span translate="I18N_ABOUT_PAGE_BREADCRUMB"></span>
-              </li>
-            </ul>
-          </navbar-breadcrumb>
-          <content>
-            <about-page></about-page>
-          </content>
-        </oppia-base-content>
-      </div>
-    </oppia-root>
+    <about-page-root></about-page-root>
```

Comment:
angular does not allow to use of transclusion in the root component unlike angularjs.
`oppia-angular-root` uses transclusion, so, it cannot be the root component.
So, I created this component two wrap `oppia-angular-root`.

This wrapper component will be added to every page while removing angularjs.
When introducing angular router, these all will be unified into single component.